app/controllers/unattended_controller.rb

```diff
@@ -55,22 +53,28 @@ def hostgroup_template
   # Generate an action for each template kind
   # i.e. /unattended/provision will render the provisioning template for the requesting host
   def host_template
+    return head(:not_found) unless params[:kind].present?
+
+    # iPXE template rendering
+    return render_intermediate_template if ipxe_request? && !@host && params[:bootstrap]
```

Comment:
I do think this group makes it slightly easier to see but I do think you could at least indent it all with `if ipxe_request?`. Then you can leave out the comment because it's self-documenting. Would that be a fair compromise?